spec/requests/admin_tasks_spec.rb

```diff
@@ -33,7 +33,7 @@
     # Compatible with claims from each policy
     Policies.all.each do |policy|
       context "with a #{policy} claim" do
-        let(:claim) { create(:claim, :submitted, policy: policy) }
+        let(:claim) { create(:claim, :submitted, policy: policy, payroll_gender: "female") }
```

Comment:
`"female"` is implicitly set by the `:submitted` trait. Do we need to explicitly set it here?